daml-script/runner/src/main/protobuf/com/daml/lf/engine/script/admin_package_service.proto

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
```

Comment:
We could add here or in the bazel rule that this is a copy of some file from canton.